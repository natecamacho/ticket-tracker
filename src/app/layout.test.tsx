import { render } from "@testing-library/react";
import Layout from "./layout";

describe("Root Layout component", () => {
  it("should render Root Layout unchanged", () => {
    const { container } = render(
      <Layout>
        <div>Test</div>
      </Layout>
    );

    expect(container).toMatchSnapshot();
  });
});
