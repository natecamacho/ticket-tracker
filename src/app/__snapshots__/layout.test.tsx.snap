// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Root Layout component should render Root Layout unchanged 1`] = `
<div>
  <html
    class="h-full"
    lang="en"
  >
    <body
      class="className h-full"
    >
      <div>
        Test
      </div>
    </body>
  </html>
</div>
`;
